File: V2/src/vanichat/config/Configuration.java

package vanichat.config;

import java.nio.charset.Charset;

public class Configuration {
    public static final Charset DEFAULT_CHARSET = Charset.forName("UTF-16");
    public static final String SERVER_IP_ADDRESS = "127.0.0.1";
    public static final int SERVER_PORT = 8080;
    public static final byte LOGIN = 0;
    public static final byte GROUP = 1;
    public static final byte MESSAGE = 2;
    public static final byte USERLINK = 3;
    public static final byte CHATLINK = 4;
    public static final String USERNAME_SEPARATOR = ":";
}

File: VaniChat C# Presentacion/VaniChat/Configuration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VaniChat
{
    public static class Configuration
    {
        public static readonly Encoding DEFAULT_CHARSET = Encoding.BigEndianUnicode;
        public static readonly string SERVER_IP_ADDRESS = "127.0.0.1";
        public static readonly int DEFAULT_PORT = 8080;
        public static readonly byte LOGIN = 0;
        public static readonly byte GROUP = 1;
        public static readonly byte MESSAGE = 2;
        public static readonly byte USERLINK = 3;
        public static readonly byte CHATLINK = 4;
        public static readonly byte RECOVERCONV = 5;
        public static readonly string USERNAME_SEPARATOR = ":";
    }
}
